packages/cloud_firestore_odm/cloud_firestore_odm_generator/cloud_firestore_odm_generator_integration_test/lib/model.dart

```diff
@@ -5,6 +5,11 @@
 // When separated accross multiple files, it is necessary to specify fromJson/toJson
 // We voluntarily don't use JsonSerializable here, as it is not supported due to
 // the generated FieldMap being private.
+import 'package:json_annotation/json_annotation.dart';
+
+part 'model.g.dart';
+
+@JsonSerializable()
```

Comment:
This is inconsistent with the comments a few lines above.

Reply:
The test is useless if we don't support not using json_serializable

Reply:
@rrousselGit Do you want me to remove this file?